content/en/docs/contribute/advanced.md

```diff
@@ -101,7 +101,7 @@ in the Kubernetes organization. The contributor's membership needs to be
 backed by two sponsors who are already reviewers.
 
 New docs contributors can request sponsors by asking in the #sig-docs channel
-on the [Kubernetes Slack instance](https://kubernetes.slack.com) or on the
```

Comment:
I prefer the original wording here. “Ask on the mailing list” is using “on”, whereas “ask in the #sig-docs” channel is using “in”.